docs/route/error-element.md

```diff
@@ -40,6 +40,12 @@ When a route does not have an `errorElement`, errors will bubble up through pare
 
 Put an `errorElement` at the top of your route tree and handle nearly every error in your app in one place. Or, put them on all of your routes and allow the parts of the app that don't have errors to continue to render normally. This gives the user more options to recover from errors instead of a hard refresh and 🤞.
 
+### Default Error Element
+
+<docs-warning>We recommend _always_ providing a at least a root-level `errorElement` before shipping your application to production, because the UI of the default `errorElement` is ugly and not intended for end-user consumption.</docs-warning>
```

Comment:
```suggestion
<docs-warning>We recommend _always_ providing at least a root-level `errorElement` before shipping your application to production, because the UI of the default `errorElement` is ugly and not intended for end-user consumption.</docs-warning>
```